ethereum/contracts/Implementation.sol

```diff
@@ -53,6 +53,7 @@ contract Implementation is Governance {
             } else if (chain == 23) { evmChainId = 42161;      // arbitrum
             } else if (chain == 24) { evmChainId = 10;         // optimism
             } else if (chain == 25) { evmChainId = 100;        // gnosis
+            } else if (chain == 30) { evmChainId = 84531;      // base
```

Comment:
We don't need this migration (and in `NftBridgeImplementation.sol`) since the `evmChainId` will already be set during the initial deployment

Reply:
True, but since it was already there, I didn't want it to be incomplete. I could delete everything if people prefer.

Reply:
Sounds like we should leave this as is then.

Reply:
Isn't this the testnet chain id though? should this be committed with all these mainnet ids? I'd lean towards omitting this for now.

Reply:
Good catch. Removing base from both core and nft implementation (didn't add it to token bridge).